src/component/elements/draggble/SVGDRaggable.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, ReactFragment } from 'react';
```

Comment:
Please fix the name of the file to SVGDraggable. Because it is just a case change please use `git mv SVGDRaggable.tsx SVGDraggable.tsx` otherwise it may be problematic

Reply:
done